test/airbrakex/exception_parser_test.exs

```diff
@@ -2,31 +2,22 @@ defmodule Airbrakex.ExceptionParserTest do
   use ExUnit.Case
 
   test "should parse exception" do
-
     exception = try do
       IO.inspect("test",[],"")
     rescue
       e -> e
     end
 
-    result = %{
-      backtrace: [
-        %{file: "(Elixir.IO) lib/io.ex", function: "inspect(\"test\", [], \"\")", line: 201},
-        %{file: "(Elixir.Airbrakex.ExceptionParserTest) test/airbrakex/exception_parser_test.exs", function: "test should parse exception/1", line: 7},
-        %{file: "(Elixir.ExUnit.Runner) lib/ex_unit/runner.ex", function: "exec_test/2", line: 253}, %{file: "(timer) timer.erl", function: "tc/1", line: 166},
-        %{file: "(Elixir.ExUnit.Runner) lib/ex_unit/runner.ex", function: "-spawn_test/3-fun-1-/3", line: 201}
-      ],
-      message: "no function clause matching in IO.inspect/3",
-      type: FunctionClauseError
-    }
-
-    %{
+    expected = %{
       backtrace: [],
       message: "no function clause matching in IO.inspect/3",
       type: FunctionClauseError
     }
+    actual = Airbrakex.ExceptionParser.parse(exception)
 
-    assert Airbrakex.ExceptionParser.parse(exception) == result
+    assert [_head | _tail] = actual.backtrace
```

Comment:
I was trying to think about the best way to implement that sort of test. The trouble is that it's going to be very brittle as per the version of Elixir used to run the tests. I thought about omitting the line portion of the backtrace entirely which makes it significantly less brittle, but it still runs into an issue with `exec_test/2` (when I run the tests locally, it backtraces to `exec_test/1` instead). I could instead do the following for tests:

```elixir
    expected = %{
      backtrace: [
        %{file: "(Elixir.IO) lib/io.ex", function: "inspect(\"test\", [], \"\")"},
        %{file: "(Elixir.Airbrakex.ExceptionParserTest) test/airbrakex/exception_parser_test.exs", function: "test should parse exception/1"},
        %{file: "(Elixir.ExUnit.Runner) lib/ex_unit/runner.ex", function: "exec_test/1"}, %{file: "(timer) timer.erl", function: "tc/1"},
        %{file: "(Elixir.ExUnit.Runner) lib/ex_unit/runner.ex", function: "-spawn_test/3-fun-1-/3"}
      ],
      message: "no function clause matching in IO.inspect/3",
      type: FunctionClauseError
    }
    actual = Airbrakex.ExceptionParser.parse(exception)
    result_backtrace = Enum.map(actual.backtrace, &%{file: &1.file, function: &1.function})

    assert result_backtrace == expected.backtrace
    assert actual.message == expected.message
    assert actual.type == expected.type
```

If you're more comfortable with that, let me know. With that change, tests remain green on my box, but I had to change the function value for the third item in the list from `exec_test/2` to `exec_test/1`.